File: 백준/Bronze/2908. 상수/상수.py
a, b = input().split()

reverse_a = "".join(reversed(a))
reverse_b = "".join(reversed(b))
reverse_a = int(reverse_a)
reverse_b = int(reverse_b)

if reverse_a > reverse_b:
  print(reverse_a)
else:
  print(reverse_b)

File: 백준/Bronze/2908. 상수/상수.cc
#include <iostream>
#include <string>
using namespace std;

int main() {
  string a, b;
  string reverse_a, reverse_b;
  cin >> a >> b;

  for (int i = 0; i < 3; i++) {
    reverse_a += a[2-i];
    reverse_b += b[2-i];
  }

  if (stoi(reverse_a) > stoi(reverse_b))
    cout << stoi(reverse_a) << "\n";
  else
    cout << stoi(reverse_b) << "\n";
}